computer.cc

```diff
@@ -1,15 +1,15 @@
 #include <iostream>
 #include "computer.h"
 #include "chessGame.h"
-
+#include "piece.h"
 
 /*
 
 IMPORTANT: The implementation for the 4 levels function is missing right now. That will be added once we have all the other components working properly.
 
 */
 
-Computer::Computer(ChessGame *subject, std::string name, vector<Piece*> * pieceArray): subject{subject}, name{name}, pieceArray{pieceArray} {
+Computer::Computer(ChessGame *subject, std::string name, std::vector<Piece*> * pieceArray): subject{subject}, name{name}, pieceArray{pieceArray} {
```

Comment:
vector of pointers to pieces and not vector of pieces

Reply:
I'll be correcting this in the next PR.